src/assets/styles/_colors_redesign.less

```diff
@@ -65,6 +65,12 @@
 @color-error-message-fade: fade(@color-error-fade, 90%);
 @color-error-message-bright-fade: fade(@color-error-text, 20%);
 
+// Button States
+@hover: rgba(0, 0, 0, 0.1);
+@focus: rgba(0, 0, 0, 0.2);
+@active: rgba(0, 0,0, 0.3);
+@disable: rgba(0, 0, 0, 0.6);
```

Comment:
Wanna re-use `@secondary-text-color` here and `fade` so `fade(@secondary-text-color, 0.1)` for hover for example